pkg/scaffold/v1/controller/controller.go

```diff
@@ -83,7 +83,8 @@ func (f *Controller) GetInput() (input.Input, error) {
 	return f.Input, nil
 }
 
-func getResourceInfo(coreGroups map[string]string, r *resource.Resource, in input.Input) (resourcePackage, groupDomain string) {
+func getResourceInfo(coreGroups map[string]string, r *resource.Resource,
```

Comment:
```suggestion
func getResourceInfo(coreGroups map[string]string, r *resource.Resource, in input.Input)
```

Reply:
The suggestion will not work. But I did a change that I hope that you liked.